source/config-file/protect-connection-strings.txt

```diff
@@ -0,0 +1,71 @@
+.. _compass-protect-connection-strings:
+.. _hide-connection-string-credentials:
+
+==========================================
+Hide Credentials in Your Connection String 
+==========================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can use the ``protectConnectionStrings`` option to hide credentials when 
+users connect to |compass-short| with a connection string. When 
+``protectConnectionStrings`` is enabled, users cannot edit hidden credentials 
```

Comment:
[suggestion]

Consider making these limitations into a bulleted list. I might also consider moving this sentence to an "About This Task" section. (see [task page prototype](https://docs-mongodbcom-staging.corp.mongodb.com/docs/docsworker-xlarge/DOCSP-23914/information-types/task/)).

For example:

"When ``protectConnectionStrings`` is enabled, users cannot perform these actions:

- Edit hidden...
- Copy the connection string...
- See the credentials...

Reply:
Good call on moving to the "About This Task" section. Updated